authors/apps/authentication/serializers.py

```diff
@@ -25,6 +25,35 @@ class Meta:
         # or response, including fields specified explicitly above.
         fields = ['email', 'username', 'password', 'token']
 
+    def validate(self, data):
+        # The `validate` method is where we make sure that the current
+        # instance of `LoginSerializer` has "valid". In the case of logging a
+        # user in, this means validating that they've provided an email
+        # and password and that this combination matches one of the users in
+        # our database.
+        username = data.get('username')
+        email = data.get('email', None)
+        password = data.get('password', None)
+
+        # Raise an exception if the username does not have atleast 3 letters
+        if not re.match("(.*[a-zA-Z]){3}", username):
```

Comment:
This is good, I suggest that username could be at least 5 characters long to increase uniqueness. We could limit to not less than 3 characters for first name and last name

Reply:
@AmosWels thanks but I think "atleast 3 letters" is enough because uniqueness is already implemented in the code.

Reply:
Alright then.